community/create-a-release.md

```diff
@@ -212,9 +212,10 @@ Docker and Docker Buildx is required for this step. Docker Buildx should be bund
 
 1. `docker login` to your own DockerHub account.
 2. Build and push docker images to your personal repository on DockerHub:
+3. MORE BUILD ARGS can be adjusted according to the performance of your computer
 
 ```shell
-docker buildx build --platform linux/amd64,linux/arm64 --tag ${your_dockerhub_username}/kvrocks:${release_version} --tag ${your_dockerhub_username}/kvrocks:latest . --output-type=registry
+docker buildx build --platform linux/amd64,linux/arm64 --tag ${your_dockerhub_username}/kvrocks:${release_version} --tag ${your_dockerhub_username}/kvrocks:latest . --output "type=registry" --build-arg MORE_BUILD_ARGS=-j12
```

Comment:
Generally, we don't need to specify `-j12` but let the platform to determinate parallelism.

Reply:
What is the meaning of `12` here? Why is `12` fit for any CPU settings? We do not need these nonsense numbers.